src/rules/noFlowFixMeComments.js

```diff
@@ -6,23 +6,23 @@ const schema = [
 
 const message = '$FlowFixMe is treated as `any` and must be fixed.';
 
-const isIdentifier = function (node, name) {
-  return node && node.type === 'Identifier' && node.name.match(name);
-};
+const isIdentifier = (node, name) => (
+  node && node.type === 'Identifier' && node.name.match(name)
+);
 
 const create = (context) => {
   const allowedPattern = context.options[0] ? new RegExp(context.options[0], 'u') : null;
-  const extraMessage = allowedPattern ? ' Fix it or match `' + allowedPattern.toString() + '`.' : '';
+  const extraMessage = allowedPattern ? ` Fix it or match \`${allowedPattern.toString()}\`.` : '';
```

Comment:
```suggestion
  const extraMessage = allowedPattern ? ` Fix it or match \'${allowedPattern.toString()}\'.` : '';
```
Maybe instead of a backtick we can use single quotations to match the format of other messages

Reply:
btw the test would then need to be updated if it is changed from backticks - _this change won't provide much value_ but we should probably have standards in the error messaging